locale/cs/LC_MESSAGES/django.po

```diff
@@ -3206,7 +3206,7 @@ msgid ""
 "Our volunteers haven't translated this article into <bdi>%(locale)s</bdi> "
 "yet. <a href=\"%(help_link)s\">Join us and help get the job done!</a>"
 msgstr ""
-"Naši dobrovolníci ještě tento článek do jazyka <bdi>%s (locale)</bdi> "
+"Naši dobrovolníci ještě tento článek do jazyka <bdi>%(locale)s</bdi> "
```

Comment:
@MikkCZ thanks for the translations, and thanks for the quick fix!

Because of the size of the MDN translation files, Pontoon only loads the first few lines. If you want to use Pontoon links in the future, I think a search term is needed to ensure the string you want is loaded when you click the page.